scarlet/observation.py

```diff
@@ -1,10 +1,101 @@
 import autograd.numpy as np
+from autograd.extend import defvjp, primitive
 
 from .frame import Frame
 from . import interpolation
 from . import fft
 from . import resampling
 
+from .bbox import Box, overlapped_slices
+
+from scarlet.operators_pybind11 import apply_filter
+
+
+def get_filter_coords(filter_values, center=None):
```

Comment:
I suggest to move this method and the next to `interpolation.py`, where we have similar functions